.github/workflows/validate.patch.yml

```diff
@@ -0,0 +1,36 @@
+name: Validate Patch
+
+# This Workflow is special and contains a workaround for a known limitation of GitHub CI.
+#
+# The problem: We don't want to run the "validate" jobs on PRs which contain only changes
+# to the docs, since these jobs take a long time to complete without providing any benefit.
+# We therefore use path-filtering in the workflow triggers for the validate jobs, namely
+# "paths_ignore: doc/**". But the "Validate post job" is a required job, therefore a PR cannot
+# be merged unless the "Validate post job" completes succesfully, which it doesn't do if we 
+# filter it out.
+#
+# The solution: We use a second job with the same name which always returns the exit code 1.
+# The logic implemented for "required" workflows accepts if 1) at least one job with that name
+# runs through, AND 2) If multiple jobs of that name exist, then all jobs of that name have to
+# finish successfully.
+#
+# Example of another repository which uses this approach:
+# - https://github.com/ZcashFoundation/zebra/tree/main/.github/workflows
+on:
+  push:
+    paths: 'doc/**'
+    branches:
+      - master
+  pull_request:
+    paths: 'doc/**'
+  release:
+    types:
+      - created
+
+jobs:
+  validate-post-job:
+    if: always()
+    name: Validate post job
+    runs-on: ubuntu-latest
+    steps:
+      - run: exit 1
```

Comment:
If all required jobs have to succeed for the run to pass, shouldn't this be `exit 0`?

Reply:
That's embarassing. You're of course completely right. I'll fix this and force-push.